test/built-ins/Temporal/Duration/compare/order-of-operations.js

```diff
@@ -264,6 +265,8 @@ const expectedOpsForZonedRelativeTo = expected.concat([
   "get options.relativeTo.timeZone.getPossibleInstantsFor",
   "call options.relativeTo.timeZone.getPossibleInstantsFor",
   "call options.relativeTo.timeZone.getOffsetNanosecondsFor",
+  // lookup
+  "get options.relativeTo.calendar.dateAdd",
```

Comment:
I wonder if expectedOpsFor{Plain,Zoned}RelativeTo could be DRYed out to better highlight actual differences.

Reply:
I've extracted the common "prefix" of both of them.